docs/api.md

```diff
@@ -212,6 +212,17 @@ gef_on_exit_unhook
 > GDB are only present in the very latest version of GDB.
 
 
+```
+@parse_arguments( {"required_argument_1": DefaultValue1, ...}, {"--optional-argument-1": DefaultValue1, ...} )
```

Comment:
Maybe show a boolean in here, e.g. `"-flag": doesntmatter`

Reply:
Why ? `flag` is a valid flag

Reply:
Then why support `-flag` and `flag`? That just makes our cli less consistent.

Reply:
And what's the point of the test for `startswith -`, then?